crates/spirv-std/src/image.rs

```diff
@@ -6,6 +6,66 @@
 mod params;
 
 pub use self::params::{ImageCoordinate, ImageCoordinateSubpassData, SampleType};
+/// A macro for creating SPIR-V `OpTypeImage` types. Always produces a
+/// [`spirv_std::image::Image<...>`](struct@Image) type.
+///
+/// The grammar for the macro is as follows:
+///
+/// ```rust
+/// # // This doctest can't use the normal macro, use an alternative version
+/// # macro_rules! Image {
+/// #   ($($tt: tt)*)=> {}
+/// # }
```

Comment:
This makes no sense to test. We should just ignore this.

Reply:
I wanted to avoid the giant *this test is ignored* warning box

But I'm happy to change it back again and just ignore it.